lib/locreator.js

```diff
@@ -44,56 +48,63 @@ Locreator.prototype.addGenericMethods = function generics() {
   };
 };
 
-Locreator.prototype.addStarMethods = function locreator(locatorId, _locator, parentWebElement) {
+Locreator.prototype.addStarMethods = function addStarMethods(locatorId, locatorJSON, parentWebElement) {
+  log('add star methods for %s', locatorId);
   var locatorObject = {};
   var drivex = this.drivex;
-  var locator = this.normify(_locator);
+  var locreator = this;
+  var locator = function () {
+    return locreator.normify(locreator.locatex(locatorJSON));
+  };
```

Comment:
Unfortunately that is exactly what this PR is about. The locale will change during the test suite lifecycle in a data driven scenario.